build-ipa.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -ex
+
+TMPDIR=$(mktemp -d -p /var/tmp)
+TMPDIR_RAMDISK=$(mktemp -d -p /var/tmp)
```

Comment:
We need to chmod this dir to 755, as it ends up as "/" in the ramdisk.

Reply:
will do, thanks!

Reply:
done